src/search/components/SearchPage.js

```diff
@@ -0,0 +1,24 @@
+import React, { PropTypes } from 'react';
```

Comment:
I wonder if this file should be rolled into the container rather than be separate? Although there's going to be pros and cons for both approaches e.g. separation + testing being easer etc...